app/controllers/questions/anonymous_intake_controller.rb

```diff
@@ -0,0 +1,5 @@
+module Questions
+  class AnonymousIntakeController < QuestionsController
+    before_action :require_intake
```

Comment:
I took a look at the source code because I was wondering if we could move require_intake definition directly into this file. It looks like `DependentsController` still inherits directly from ApplicationController and uses the `before_action :require_intake`. I know that controller acts a bit differently than others in the flow, but I'm wondering if it should/can inherit from the AuthenticatedIntakeController instead?

(Because the definition of require_intake calls code that looks for the intake either from the current_client or the session intake, it seems like the code path would be "safe" even without the refactor of DependentsController!)

Reply:
You are right to call out the `:require_intake` in dependentsController! It should have the same client login behavior as the AuthenticatedIntakeController. However, I didn't want it to inherit all the QuestionsController stuff, so instead I treated it similar to a portal controller, and used the client access control concern

```ruby
class DependentsController < ApplicationController
  include ClientAccessControlConcern

  before_action :require_client_login
```